docroot/themes/humsci/humsci_basic/humsci_basic.libraries.yml

```diff
@@ -1,3 +1,8 @@
 base:
   js:
-    src/js/build/scripts.js: {}
+    scripts/build/scripts.js: {}
+
+no-js:
+  header: true
+  js:
+    scripts/no-js.js: {}
```

Comment:
a `no-js` javascript file? this doesn't make sense

Reply:
Yeah it would make more sense to call this something like `js-class.js` if we end up keeping it. 😄

Reply:
This was removed as part of the modernizr recommendation. 👍